topics/proteomics/tutorials/metaquantome-data-creation/tutorial.md

```diff
@@ -0,0 +1,656 @@
+---
+layout: tutorial_hands_on
+
+title: metaQuantome Data creation tutorial
+zenodo_link: '(https://doi.org/10.5281/zenodo.4037137)'
```

Comment:
I think the parentheses here are failing the linting check. Suggest changing the line to:
```
zenodo_link: "https://doi.org/10.5281/zenodo.4037137"
```